### DISCUSSION TITLE
Clarification on the object detection tutorial

### DISCUSSION THREAD

**Discussion by gshikhri** [sep-developers/sep#89] [Q&A]
Hi, 
I am a novice user of Source Extractor and stumbled upon SEP accidentally. I was going through the object detection tutorial that uses sep.extract. While visualizing the object detection using matplotlib Ellipses, I noticed that the object property 'a' and 'b' were scaled by a factor of 6. See below: 

```python
e = Ellipse(xy=(objects['x'][i], objects['y'][i]),
                width=6*objects['a'][i],
                height=6*objects['b'][i],
                angle=objects['theta'][i] * 180. / np.pi)
```

I was wondering if someone here could be able to tell me why do we need to scale the semi-major and semi-minor axis by a factor of 6?

Best regards,
Shikhar

**Comment by kbarbary** [comment]
Good question! The scaling by `6` is arbitrary and is just for the purpose of drawing ellipses that are a pleasing visual size.

The `a` and `b` parameters are the root-mean-square (RMS) of the profile along the major and minor axes respectively, and it just happens that the RMS is pretty small compared to the apparent size of the object in the image. So we scale the ellipse up to encompass all the object's significant pixels.
